frontend/src/Pages/AllUser/HomePage/HeroSection/Hero.jsx

```
@@ -0,0 +1,27 @@
+                style={{
+                    backgroundImage: "url(https://i.ibb.co.com/JRs9429M/Whats-App-Image-2025-02-10-at-11-00-20-080a95b3.jpg)",
+                }}>
+                <div className="hero-overlay bg-opacity-60"></div>
+                <div className="hero-content text-neutral-content text-center">
+                    <div className="max-w-md">
+                        <h1 className="mb-5 text-5xl font-bold">Hello there</h1>
+                        <p className="mb-5">
+                            Provident cupiditate voluptatem et in. Quaerat fugiat ut assumenda excepturi exercitationem
+                            quasi. In deleniti eaque aut repudiandae et a id nisi.
+                        </p>
+                        <button className="btn btn-primary">Get Started</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)